requirements.txt

```diff
@@ -24,6 +24,7 @@ oslo.config<1.13,>=1.12.1
 oslo.utils<3.1.0
 paramiko<2.2,>=2.1.2
 passlib==1.6.5
+prance
```

Comment:
Do we need to add corresponding entry to some component in-requirements.txt and fixed-requirements.txt since this file is auto-generated?